straxen/contexts.py

```diff
@@ -261,56 +281,88 @@ def xenonnt_simulation(output_folder='./strax_data',
                     **straxen.contexts.xnt_common_config,),
         **straxen.contexts.xnt_common_opts, **kwargs)
     st.register(wfsim.RawRecordsFromFaxNT)
+    st.apply_cmt_version(f'global_{cmt_version}')
 
     if _forbid_creation_of is not None:
         st.context_config['forbid_creation_of'] += strax.to_str_tuple(_forbid_creation_of)
 
-    st.apply_cmt_version(f'global_{cmt_version}')
-    if not cmt_run_id:
-        raise ValueError('You have to specify a run_id which should be used to initialize '
-                         'the corrections.')
-    if _cmt_run_id_proc_only is None:
-        _cmt_run_id_proc_only = cmt_run_id
+    # doing sanity checks for cmt run ids for simulation and processing
+    if not (cmt_run_id_sim is None) and not (cmt_run_id_proc is None):
+        if not (cmt_run_id_sim==cmt_run_id_proc):
+            print("INFO : divergent CMT runs for simulation and processing")
+            print("    cmt_run_id_sim".ljust(25), cmt_run_id_sim)
+            print("    cmt_run_id_proc".ljust(25), cmt_run_id_proc)
+    elif (cmt_run_id_sim is None) and not (cmt_run_id_proc is None):
+        cmt_run_id_sim=cmt_run_id_proc
+    elif not (cmt_run_id_sim is None) and (cmt_run_id_proc is None):
+        cmt_run_id_proc=cmt_run_id_sim
+    else:
+        raise RuntimeError("Trying to set both cmt_run_id_sim and cmt_run_id_proc to None")
 
-    # Setup processing
     # Replace default cmt options with cmt_run_id tag + cmt run id
     cmt_options = straxen.get_corrections.get_cmt_options(st)
+
+    # First, fix gain model for simulation
+    st.set_config({'gain_model_mc': tuple(['cmt_run_id',
+                                           cmt_run_id_sim,
+                                           *cmt_options['gain_model']])})
```

Comment:
```suggestion
    st.set_config({'gain_model_mc': ('cmt_run_id', cmt_run_id_sim, *cmt_options['gain_model'])})
```

Reply:
I think having it aligned like that actually improves the readability ( i mean the spaces before/after `=`)